### DISCUSSION TITLE
Dual Booting with Mocaccino?

### DISCUSSION THREAD

**Discussion by VolvoBirkenstock** [mocaccinoOS/mocaccino#82] [Q&A] [answered]
Hi, Mocaccino (and Sabayon) Gurus.  

I think I have two questions, but many more may follow depending on your responses. 

**1.** Has anyone migrated from Sabayon to Mocaccino on a dual-boot system where the non-Sabayon system is Windows?  I ask because once I migrate from Sabayon to Mocaccino, I _must_ have the ability to dual-boot to Windows.  There is software on the Windows partition that is irreplaceable, so that Windows partition must remain bootable.  

**2.** Has anyone built a new Mocaccino system that is also dual-bootable?  I ask this question in the event that I cannot cleanly merge from Sabayon to Mocaccino.  

A little background to the above questions follows:  I already tried dual-booting on a system I set up as an experiment, and I could not determine how to do it using the Calamares installer to perform this task, so if this is doable, then I would greatly appreciate a cookbook procedure.  I was able to install a single-boot Mocaccino system, and then noticed that grub-mkconfig existed, but grub2-mkconfig did not exist; It might just be a matter of installing the grub2 package.  I am under the assumption that I needed to use grub2-mkconfig for finally configuring the system for dual-boot when I was running Sabayon in a dual-boot configuration.  I have been installing Mocaccino while using the MocaccinoOS-KDE-0.20220515.iso image.  

Thank you very much in advance.

**Comment by joostruis** [comment, marked as answer]
Do not try to migrate. It is totally unreliable.
Within Gentoo grub2* programs got renamed to grub* so it is excactly the same thing.

If your PC has such an important partition I would not recommend using MocaccinoOS at all. It is beta software.

**Reply by VolvoBirkenstock** [reply]
Thank you very much, Joost.  

I know this is likely a stupid question, but I will ask it because I am me. ;-)  
I suspect there may be too many unknowns for you to answer with what you consider to be an accurate estimate. 
How much more time do you think is required before you anticipate moving beyond the beta phase?  

Again, thank you very much!

**Reply by joostruis** [reply]
Hard to answer.
But even when we get passed the beta point I fear I can't guarantee that migrating a system from Sabayon to MocaccinoOS will work in a reliable fashion. (In what I consider reliable I must say)

We really need some feedback from users to get passed the beta point right now.

**Reply by VolvoBirkenstock** [reply]
OK, Joost, I'll do what I can, but time is pretty tight for me right now.  I need to learn more about packages; how to call them and if multiple packages can be called using a hierarchy for the same result as is achieved using a meta-package approach.  I'd also like to learn more about how to determine what functionality is inside each package.  I never was a guru with Sabayon, but with a lot of time and a lot of questions directed at the gurus I was able to achieve good results.  

Should I assume that you're no longer interested in tests involving solely the "stable" packages, but are much more interested in testing packages that are not in the "stable" repositories?  That is, should I be building systems using the unstable repositories and giving feedback from that? 

Do you need to test across a hardware spectrum?  Should I assume that Virtual Machines used as a test-bed might not provide accurate results?  I suppose I could continue my attempts at dual-booting on the system I mentioned earlier...
